src/theme.c

```diff
@@ -213,6 +213,7 @@ theme_builtin(struct theme *theme)
 {
 	theme->border_width = 1;
 	theme->padding_height = 3;
+	theme->title_height = 0;
```

Comment:
Suggest setting this to `INT_MIN` (representing unset) and then fallback to the default in `post_processing()` if not set.